CONTRIBUTING.md

```diff
@@ -21,6 +21,8 @@ reported the issue. Please try to include as much information as you can. Detail
 
 
 ## Contributing via Pull Requests
+**Before start; please do follow the [DEVELOPER Guidlines](./docs/DEVELOPER.md) first.**
```

Comment:
could be much more prominent
```suggestion

**Read [DEVELOPER Guidelines](./docs/DEVELOPER.md) first.**
```

I would even repeat this at the top of CONTRIBUTING.md. It's quite important and much more useful that the boilerplate that is currently in CONTRIBUTING.md. (p.s.: can CONTRIBUTING.md be reduced? is all of its text essential and useful for most devs? if not, they may miss the actually important parts)